src/backend_vulkan.zig

```diff
@@ -25,15 +25,15 @@ pub const ImGui_ImplVulkan_InitInfo = extern struct {
     msaa_samples: u32 = 0, // vkSampleCountFlags
 
     // Optional fields
-    pipeline_cache: VkHandle = 0, // VkPipelineCache
+    pipeline_cache: VkHandle = null, // VkPipelineCache
     subpass: u32 = 0,
     descriptor_pool_size: u32 = 0,
 
     use_dynamic_rendering: bool = false,
     pipeline_rendering_create_info: VkPipelineRenderingCreateInfo = .{},
 
-    allocator: ?*const anyopaque = null,
-    check_vk_result_fn: ?*const fn (err: u32) void = null,
+    allocator: VkHandle = null,
```

Comment:
`allocator` should really be `?*const VkAllocationCallbacks` but let's just leave it as `?*const anyopaque` for now.